package.json

```diff
@@ -52,9 +54,16 @@
     "@types/sharp": "^0.25.0",
     "@types/styled-components": "^4",
     "@types/styled-system": "^5.1.10",
-    "dotenv-cli": "^4.0.0",
+    "dotenv": "^8.2.0",
     "next-transpile-modules": "^4.1.0",
+    "ts-node": "^9.0.0",
     "typescript": "^4.0.5",
     "typescript-styled-plugin": "^0.15.0"
+  },
+  "_moduleAliases": {
+    "lodash-es": "node_modules/lodash",
+    "utils": "src/utils",
+    "components": "src/components",
+    "data": "src/data"
   }
```

Comment:
This is required by the `module-alias` module. I added this so that I could use `ts-node` to run some of the scripts like `prime-cache`.

Reply:
Nice package!